### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by melissawm**
# Description
Simple maintenance PR to update .gitignore file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -139,24 +132,25 @@ windows/
...
+docs/jupyter_execute/
+docs/sg_execution_times.rst
+
 # come from npe2 docs
 docs/plugins/_npe2_*.md
-
-napari/settings/napari.schema.json
-
-docs/jupyter_execute/
-docs/.jupyter_cache/
```
Comment (lucyleeow) [inline .gitignore:161]: Was this not needed anymore?

**Review by lucyleeow — APPROVED** [review]
Naive review, but LGTM!

File: .gitignore (hunk 1 above)
Comment (melissawm) [reply .gitignore:161]: You are right, the jupyter_execute still needs to be there. I just moved it to the "documentation" section. The .jupyter_cache folder really doesn't get created with our current workflow but I'm happy to add it back as it really doesn't hurt 😄

**Review by Czaki — APPROVED** [review]

**Comment by melissawm** [comment]
Thanks for the reviews - merging after discussion with @Czaki. Cheers!

**Pull request merged**